atomic_reactor/schemas/fetch-artifacts-nvr.json

```diff
@@ -0,0 +1,56 @@
+{
+  "$schema": "http://json-schema.org/draft-04/schema#",
+
+  "title": "Artifacts to be fetched from Koji builds.",
+
+  "type": "array",
+
+  "items": {
+
+    "type": "object",
+
+		"description": "A specific Koji buid and archives to be fetched.",
+
+    "properties": {
+
+      "nvr": {
+        "description": "The Name-Version-Release of a Koji build.",
+        "type": "string"
+      },
+
+      "archives": {
+
+        "type": "array",
+
+        "items": {
+
+          "type": "object",
+
+          "description": "Matcher for archives in Koji build.",
+
+          "properties": {
+
```

Comment:
My futile approach into making this file more readable. Changed it so it's more consistent with config.json.